front_office/account_page/templates/login.html

```diff
@@ -1,18 +1,19 @@
-{% extends "base.html" %}
+{% extends "front_base.html" %}
```

Comment:
this is a front-office login page, the base for this one should stay as a base.html, there will be a separate login for backoffice

Reply:
Since there will be 2 of base.html they should both be named differently. as in , front_/back_. But Okay.